src/microbe_stage/SpawnedRigidBody.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Godot;
+
+/// <summary>
+///   All nodes that can be spawned with the spawn system must derivative from this class
```

Comment:
```suggestion
///   All nodes that can be spawned with the spawn system must be a derivative from this class
```

And perhaps we'd even prefer another term like "must inherit this class"?